src/lib/check-flags.js

```diff
@@ -0,0 +1,13 @@
+export default (item, newItem) => {
+    // check flags
+    const itemFlags = item.flags ? item.flags : false
+    const newItemFlags = newItem.flags ? newItem.flags : false

```

Comment:
esses ternários não têm utilidade aqui 😬 

```suggestion
    const itemFlags = item.flags
    const newItemFlags = newItem.flags
```

Na verdade nem vejo vantagem em declarar essas constantes não (o nome é quase tão grande quanto, nem reduziu o código), poderia só usar `item.flags` e `newItems.flags` mesmo...

Reply:
Mandar direto né. Pensei que ia reclamar porque ficaria repetindo eles kkk

Reply:
Repetir variável ? 🙄 

Mas `const itemFlags = item.flags` não seria problema não, assim como não seria problema usar `item.flags`..
Só esse ternário mesmo (que inclusive já te vi fazer em outros lugares) que no fim tá sendo um if e complexidade a mais sem nenhuma razão de ser 🙃

Reply:
Quando testei aqui, deu pau, quando era undefined, no navegador acusou que era undefined, por isso, coloquei o ternário. Valeu